data_structures/src/builders.rs

```diff
@@ -27,7 +27,7 @@ pub const PROTOCOL_VERSION: u32 = 0x0000_0001;
 pub const CAPABILITIES: u64 = 0x0000_0000_0000_0001;
 
 /// User agent
-pub const USER_AGENT: &str = "full-node-desktop-edition";
+pub const USER_AGENT: &str = "full-node-desktop-edition-v1.4.1";
```

Comment:
Let's construct this string dinamically with `witnet-rust`, plus arch, plus OS, plus version!